### ISSUE TITLE
Is the Video.js adapter supposed to allow for controlling iframes with videojs?

### ISSUE THREAD

**Issue by DoomTay** [embedly/player.js#69]
http://jsfiddle.net/q815knga/7/

I thought that the adapter would allow for controlling an embedded iframe through player.js. Am I looking at this all wrong?

**Comment by jerlam06** [comment]
Wonder the same...

**Comment by Xaekai** [comment]
No, if you are using an embedded iframe, vjs doesn't enter the picture.

http://jsfiddle.net/j8e0y2k6/
